[Bug]: New issues does not get added to project

### Priority

MAJOR

### Actual behaviour

Issues and PRs should get added to the project.

### Expected behaviour

They don't -> either they don't trigger the action or the action fails.

### Steps to reproduce

1. Create an issue
2. See that it does not get added to the project
3. Check if the action has run

### What platform are you seeing the problem on?

Other

### Relevant log output

```shell
Run item_id="$( gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
  item_id="$( gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
    mutation($project:ID!, $pr:ID!) {
      addProjectNextItem(input: {projectId: $project, contentId: $pr}) {
        projectNextItem {
          id
        }
      }
    }' -f project=$PROJECT_ID -f pr=$PR_ID --jq '.data.addProjectNextItem.projectNextItem.id')"
  
  echo 'ITEM_ID='$item_id >> $GITHUB_ENV
  shell: /usr/bin/bash -e {0}
  env:
    PROJECT_ID: "MDExOlByb2plY3ROZXh0MzExMw=="
    DATE_FIELD_ID: "MDE2OlByb2plY3ROZXh0RmllbGQyODcxNw=="
    STATUS_FIELD_ID: "MDE2OlByb2plY3ROZXh0RmllbGQyNTg1NQ=="
    TODO_OPTION_ID: "2c597098"
    GITHUB_TOKEN: ***
    PR_ID: 
gh: Could not resolve to a node with the global id of ''
Error: Process completed with exit code 1.
```


### Attatchments

![image](https://user-images.githubusercontent.com/39928082/133326942-14fd7729-d6d9-4447-ae53-cf8fd04e8629.png)


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.github/workflows/auto_add_issue_to_project.yml
+++ b/.github/workflows/auto_add_issue_to_project.yml
@@ -1,0 +1,71 @@
+name: Add Issue to project
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add_item_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+          ORGANIZATION: BHelpful
+          PROJECT_NUMBER: 4
+        run: |
+          gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
+                  id
+                  fields(first:20) {
+                    nodes {
+                      id
+                      name
+                      settings
+                    }
+                  }
+                }
+              }
+            }' -f org=$ORGANIZATION -F number=$PROJECT_NUMBER > project_data.json
+          echo 'PROJECT_ID='$(jq '.data.organization.projectNext.id' project_data.json) >> $GITHUB_ENV
+          echo 'DATE_FIELD_ID='$(jq '.data.organization.projectNext.fields.nodes[] | select(.name== "Date posted") | .id' project_data.json) >> $GITHUB_ENV
+      - name: Add Issue to project
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+          ISSUE_ID: ${{ github.event.issue.node_id }}
+        run: |
+          item_id="$( gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            mutation($project:ID!, $issue:ID!) {
+              addProjectNextItem(input: {projectId: $project, contentId: $issue}) {
+                projectNextItem {
+                  id
+                }
+              }
+            }' -f project=$PROJECT_ID -f issue=$ISSUE_ID --jq '.data.addProjectNextItem.projectNextItem.id')"
+          echo 'ITEM_ID='$item_id >> $GITHUB_ENV
+      - name: Get date
+        run: echo "DATE=$(date +"%Y-%m-%d")" >> $GITHUB_ENV
+
+      - name: Set fields for Issue
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+        run: |
+          gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            mutation (
+              $project: ID!
+              $item: ID!
+              $date_field: ID!
+              $date_value: String!
+            ) {
+              set_date_posted: updateProjectNextItemField(input: {
+                projectId: $project
+                itemId: $item
+                fieldId: $date_field
+                value: $date_value
+              }) {
+                projectNextItem {
+                  id
+                }
+              }
+            }' -f project=$PROJECT_ID -f item=$ITEM_ID -f date_field=$DATE_FIELD_ID -f date_value=$DATE --silent

--- a/.github/workflows/auto_add_pr_to_project.yml
+++ b/.github/workflows/auto_add_pr_to_project.yml
@@ -1,0 +1,71 @@
+name: Add PR to project
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add_item_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+          ORGANIZATION: BHelpful
+          PROJECT_NUMBER: 4
+        run: |
+          gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
+                  id
+                  fields(first:20) {
+                    nodes {
+                      id
+                      name
+                      settings
+                    }
+                  }
+                }
+              }
+            }' -f org=$ORGANIZATION -F number=$PROJECT_NUMBER > project_data.json
+          echo 'PROJECT_ID='$(jq '.data.organization.projectNext.id' project_data.json) >> $GITHUB_ENV
+          echo 'DATE_FIELD_ID='$(jq '.data.organization.projectNext.fields.nodes[] | select(.name== "Date posted") | .id' project_data.json) >> $GITHUB_ENV
+      - name: Add PR to project
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+          PR_ID: ${{ github.event.pull_request.node_id }}
+        run: |
+          item_id="$( gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            mutation($project:ID!, $pr:ID!) {
+              addProjectNextItem(input: {projectId: $project, contentId: $pr}) {
+                projectNextItem {
+                  id
+                }
+              }
+            }' -f project=$PROJECT_ID -f pr=$PR_ID --jq '.data.addProjectNextItem.projectNextItem.id')"
+          echo 'ITEM_ID='$item_id >> $GITHUB_ENV
+      - name: Get date
+        run: echo "DATE=$(date +"%Y-%m-%d")" >> $GITHUB_ENV
+
+      - name: Set fields for Issue
+        env:
+          GITHUB_TOKEN: ${{secrets.MY_GITHUB_TOKEN}}
+        run: |
+          gh api graphql --header 'GraphQL-Features: projects_next_graphql' -f query='
+            mutation (
+              $project: ID!
+              $item: ID!
+              $date_field: ID!
+              $date_value: String!
+            ) {
+              set_date_posted: updateProjectNextItemField(input: {
+                projectId: $project
+                itemId: $item
+                fieldId: $date_field
+                value: $date_value
+              }) {
+                projectNextItem {
+                  id
+                }
+              }
+            }' -f project=$PROJECT_ID -f item=$ITEM_ID -f date_field=$DATE_FIELD_ID -f date_value=$DATE --silent

